### ISSUE TITLE
(Suggestion) Eatable hay bales/Placeable/Stackable wheat?

### ISSUE THREAD

**Issue by NavyT** [capnkirok/animania#415]
Could sheep/horses/animals that eat wheat
be able to eat hay bales? Or wheat could be placeable? Maybe the wheat could even be stackable. And slowly go's down when animals eat it.

**Comment by Tschipp** [comment]
> Maybe the wheat could even be stackable. And slowly go's down when animals eat it.

That's basically the functionality of troughs

**Comment by NavyT** [comment]
Oh. I was thinking like how the seeds work when you place them down. Just a suggestion of the other way. Two options on how you could place them.

**Comment by Wolfixsp** [comment]
i agree like when im going adventure for a time and need to my cows be well fed and happy

**Comment by BryanBoru** [comment]
This is especially significant if you have neighbors close enough to keep your chunk loaded while out adventuring.
